datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -601,7 +601,7 @@ public Pair<SdxCluster, FlowIdentifier> resizeSdx(final String userCrn, final St
         return Pair.of(sdxCluster, flowIdentifier);
     }
 
-    public SdxCluster refreshDataHub(String clusterName, String datahubName) {
+    public List<String> refreshDataHub(String clusterName, String datahubName) {
```

Comment:
Recommend adding a unit test for this just to ensure that it won't be broken in the future.